contrib/coreos/user-data.example

```diff
@@ -3,7 +3,7 @@
 coreos:
   etcd2:
     # generate a new token for each unique cluster from https://discovery.etcd.io/new
-    discovery: #DISCOVERY_URL
+    discovery: $DISCOVERY_URL
```

Comment:
This change will break [`make discovery-url`](https://github.com/deis/deis/blob/master/Makefile#L37), which is how  other providers get an `etcd` discovery URL.